### PR TITLE
760 Use TDP 1.0 releases files

### LINKED ISSUE #706: spark3: spark History Server fails to start
```
2023-02-21 13:26:08,629 - INFO  [main:SslContextFactory@358] - x509=X509@36bc415e(mehdi-master-03.novalocal,h=[mehdi-master-03.novalocal],a=[/172.16.0.153],w=[]) 
for Server@6a714237[provider=null,keyStore=file:///etc/ssl/certs/keystore.jks,trustStore=null]

2023-02-21 13:26:08,640 - ERROR [main:Logging@94] - Failed to bind HistoryServer
java.security.UnrecoverableKeyException: Password must not be null
        at sun.security.provider.JavaKeyStore.engineGetKey(JavaKeyStore.java:134)
        at sun.security.provider.JavaKeyStore$JKS.engineGetKey(JavaKeyStore.java:57)
        at sun.security.provider.KeyStoreDelegator.engineGetKey(KeyStoreDelegator.java:96)
        at sun.security.provider.JavaKeyStore$DualFormatJKS.engineGetKey(JavaKeyStore.java:71)
        at java.security.KeyStore.getKey(KeyStore.java:1023)
        at sun.security.ssl.SunX509KeyManagerImpl.<init>(SunX509KeyManagerImpl.java:145)
        at sun.security.ssl.KeyManagerFactoryImpl$SunX509.engineInit(KeyManagerFactoryImpl.java:70)
        at javax.net.ssl.KeyManagerFactory.init(KeyManagerFactory.java:256)
        at org.sparkproject.jetty.util.ssl.SslContextFactory.getKeyManagers(SslContextFactory.java:1249)
        at org.sparkproject.jetty.util.ssl.SslContextFactory$Server.getKeyManagers(SslContextFactory.java:2364)
        at org.sparkproject.jetty.util.ssl.SslContextFactory.load(SslContextFactory.java:373)
        at org.sparkproject.jetty.util.ssl.SslContextFactory.doStart(SslContextFactory.java:244)
        at org.sparkproject.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
        at org.sparkproject.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
        at org.sparkproject.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
        at org.sparkproject.jetty.server.SslConnectionFactory.doStart(SslConnectionFactory.java:97)
        at org.sparkproject.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
        at org.sparkproject.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
        at org.sparkproject.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
        at org.sparkproject.jetty.server.AbstractConnector.doStart(AbstractConnector.java:321)
        at org.sparkproject.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:81)
        at org.sparkproject.jetty.server.ServerConnector.doStart(ServerConnector.java:234)
        at org.sparkproject.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
        at org.apache.spark.ui.JettyUtils$.newConnector$1(JettyUtils.scala:303)
        at org.apache.spark.ui.JettyUtils$.sslConnect$1(JettyUtils.scala:322)
        at org.apache.spark.ui.JettyUtils$.$anonfun$startJettyServer$4(JettyUtils.scala:326)
        at org.apache.spark.ui.JettyUtils$.$anonfun$startJettyServer$4$adapted(JettyUtils.scala:326)
        at org.apache.spark.util.Utils$.$anonfun$startServiceOnPort$2(Utils.scala:2383)
        at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:158)
        at org.apache.spark.util.Utils$.startServiceOnPort(Utils.scala:2375)
        at org.apache.spark.ui.JettyUtils$.$anonfun$startJettyServer$2(JettyUtils.scala:326)
        at org.apache.spark.ui.JettyUtils$.$anonfun$startJettyServer$2$adapted(JettyUtils.scala:315)
        at scala.Option.map(Option.scala:230)
        at org.apache.spark.ui.JettyUtils$.startJettyServer(JettyUtils.scala:315)
        at org.apache.spark.ui.WebUI.bind(WebUI.scala:147)
        at org.apache.spark.deploy.history.HistoryServer.bind(HistoryServer.scala:164)
        at org.apache.spark.deploy.history.HistoryServer$.main(HistoryServer.scala:310)
        at org.apache.spark.deploy.history.HistoryServer.main(HistoryServer.scala)
```

### LINKED ISSUE #760: Use TDP 1.0 releases files

### PR DESCRIPTION
**Pull request by PACordonnier**
#### Which issue(s) this PR fixes

Fixes #760 

#### Additional comments

Still as draft due to spark not being release yet

#### Agreements

- [x] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Review by mehdibn — APPROVED** [review]

File: tdp_vars_defaults/spark/spark.yml [hunk 1]
```diff
@@ -4,9 +4,9 @@
 ---
 # Spark version
 spark_version: spark
-spark_release: spark-2.3.5-TDP-0.1.0-SNAPSHOT-bin-tdp
+spark_release: spark-2.3.4-1.0-bin-tdp
 spark_dist_file: "{{ spark_release }}.tgz"
-spark_hbase_dist_file: hbase-spark-2.3.5-1.0.0-TDP-0.1.0-SNAPSHOT.jar
+spark_hbase_dist_file: hbase-spark-1.0.1-SNAPSHOT.jar
```
Comment (rpignolet) [inline tdp_vars_defaults/spark/spark.yml:9]: The JAR name is wrong. See https://github.com/TOSIT-IO/hbase-connectors/releases/tag/2.3.4-1.0.0-0.0.
```suggestion
spark_hbase_dist_file: hbase-spark-2.3.4-1.0.0-0.0.jar
```

**Comment by rpignolet** [comment]
`hbase_hbck2_release` have the TDP SNAPSHOT version.

File: tdp_vars_defaults/hbase/hbase.yml [hunk 2]
```diff
@@ -3,11 +3,11 @@
 
 ---
 # HBase version
-hbase_release: hbase-2.1.10-TDP-0.1.0-SNAPSHOT
+hbase_release: hbase-2.1.10-1.0
 hbase_dist_file: "{{ hbase_release }}-bin.tar.gz"
 
 # Hbck2 version
-hbase_hbck2_release: hbase-hbck2-1.1.0-TDP-0.1.0-SNAPSHOT 
+hbase_hbck2_release: hbase-hbck2-1.1.0-0.0.jar
```
Comment (rpignolet) [inline tdp_vars_defaults/hbase/hbase.yml:10]: Do not put `.jar` here.
```suggestion
hbase_hbck2_release: hbase-hbck2-1.1.0-0.0
```

Comment (PACordonnier) [reply tdp_vars_defaults/hbase/hbase.yml]: Done in latest push

**Review by rpignolet — APPROVED** [review]

**Pull request merged**